docs/topics/install-olympia/installation.rst

```diff
@@ -36,6 +36,8 @@ if you're running a recent version, you can `install them automatically
 
     sudo aptitude install python-dev python-virtualenv libxml2-dev libxslt1-dev libmysqlclient-dev memcached libssl-dev swig openssl curl libjpeg-dev zlib1g-dev libsasl2-dev
 
+**Note**: 
```

Comment:
You could use the same syntax (`.. note::`) as below in the file, on line 204, what do you think?